pkg/charts/parse.go

```diff
@@ -79,6 +79,10 @@ func GetPackage(rootFs billy.Filesystem, name string) (*Package, error) {
 	if err != nil {
 		return nil, err
 	}
+	// version and packageVersion can not exist at the same time although both are optional
+	if packageOpt.Version != "" && packageOpt.PackageVersion != 0 {
+		return nil, fmt.Errorf("Cannot have both version and packageVersion at the same time")
+	}
```

Comment:
The fact that the `PackageVersion` can no longer be specified as 0 seems like a breaking change to users since this was previously allowed.

Instead of having the `packageOpt.PackageVersion` be an int, can you change it to a pointer to an int? That way, the omitempty logic you have below would also work.

Reply:
Wd should also change `version` to be a pointer, since it is also an optional field that can be nil. That way, this statement just checks whether both are non-nil.